### PR TITLE
Validated delete functionality of label, in case of node having single label

### PR DESCRIPTION
**Pull request by Priyankakore21**

### CODE REVIEW

File: src/browser/modules/EditorInfo/DisplayNodeDetails.jsx [hunk 1]
```diff
@@ -62,6 +62,7 @@ const LabelSection = props => {
                     <tbody>
                       <tr>
                         <DisplayLabel
+                          isSingleLabel={labels.length > 1}
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/DisplayNodeDetails.jsx]: reword to isDeletable

**Review by rahul-tarafdar-synerzip — CHANGES_REQUESTED** [review]
Rest LGTM

File: src/browser/modules/EditorInfo/DisplayNodeDetails.jsx (hunk 1 above)
Comment (Priyankakore21) [reply src/browser/modules/EditorInfo/DisplayNodeDetails.jsx]: Done with changes

**Review by rahul-tarafdar-synerzip — APPROVED** [review]
LGTM

**Pull request merged**